apiServer/taas/utils/torpedoutils.go

```diff
@@ -177,44 +178,56 @@ func CollectSupport(c *gin.Context) {
 // ScheduleAppsAndValidate : This API schedules multiple applications on the cluster and validates them
 // context is created as a global context to be accessed later in further tests
 func ScheduleAppsAndValidate(c *gin.Context) {
+
+	var requestBody struct {
+		NamespaceSuffix string   `json:"nsSuffix"`
+		AppList         []string `json:"appList"`
+	}
 	var errors []error
+	errStrings := make([]string, 0)
 	errChan := make(chan error, 100)
 	if !checkTorpedoInit(c) {
 		c.JSON(http.StatusInternalServerError, gin.H{
-			"error": fmt.Errorf("Error happened while doing InitInstance()"),
+			"error": fmt.Errorf("error happened while doing InitInstance()"),
```

Comment:
nit: change it to `error during InitInstance`

Reply:
done